src/test/java/com/google/javascript/clutz/untyped_provide.js

```diff
@@ -0,0 +1,10 @@
+goog.provide('untyped.Provide');
+// Surprisingly this behaves differently from undefined_provide.js, there's no symbol at all.
+
+/**
+ * Has no type, but interesting structure.
+ */
+untyped.Provide = {
+  SOME_FIELD: 'x',
+  SOME_OTHER_FIELD: 'y'
```

Comment:
This guys are still unaccessible in TS, how about we investigate emitting any for them (doesn't need to be in this PR).